Re-organise 'Contributing' section (#318)
napari/docs: docs/developers/index.md

(contributing-index)=
# Contributing

We welcome your contributions! Here you will find resources to help you contribute
to napari.

napari has a [Code of Conduct](napari-coc) that should be honored by everyone who participates in the `napari` community.

- [](contact): how to contact us.
- [](roadmaps): convey the general direction for napari development and possible
  work plans.
- **Contributor guides**: resources on for contributors:
  - [Contributing guide](napari-contributing)
    - [Development installation](dev-installation)

>     - [Translations](translations)
>     - [Testing](napari-testing)

Reviewer: ```suggestion
    - [Testing](napari-testing)
    - [Translations](translations)
```
matching order in ToC

    - [Performance](performance)
      - [Profiling](profiling)
      - [Benchmarks](napari-benchmarks)
    - [Contributing documentation](docs_contributing_guide)
  - [](architecture-index)

- **Core developer guides**: resources for core developers, including information for
  onboarding new core developers:

>   - [Core Developer guide](./core_dev_guide)

Reviewer: ```suggestion
  - [Core Developer guide](core-developer-guides)
```

>   - [Maintenance](./maintenance)

Reviewer: ```suggestion
  - [Maintenance](coredev/maintenance)
```
Or, better yet add a reference in maintenance.md?

Author: There's already a reference!

>   - [Release guide](./release)

Reviewer: ```suggestion
  - [Release guide](coredev/release)
```
Same as above

  - [Packaging](napari-packaging)
  - [Deploying documentation](docs-deployment)

(contact)=

## Contact

There are a number of ways to contact the napari community:

- [zulip](https://napari.zulipchat.com/): the zulip napari community chat channel.
- [GitHub issue](https://github.com/napari/napari/issues): feel free to open an
  issue in our GitHub repository. Issues and pull-requests are written in [Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/about-writing-and-formatting-on-github). You can find a comprehensive guide [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
- [imagesc forum](https://forum.image.sc/tags/napari): napari is a community partner

>   on the imagesc forum. All usage support requests should be posted on the forum with

Reviewer: ```suggestion
  on the Image.sc forum. All usage support requests should be posted on the forum with
```
